addons/hr/models/res_users.py

```diff
@@ -149,7 +149,7 @@ def _employee_ids_domain(self):
     employee_type = fields.Selection(related='employee_id.employee_type', readonly=False, related_sudo=False)
     employee_resource_calendar_id = fields.Many2one(related='employee_id.resource_calendar_id', string="Employee's Working Hours", readonly=True)
 
-    create_employee = fields.Boolean(store=False, default=False, copy=False, string="Technical field, whether to create an employee")
+    create_employee = fields.Boolean(store=False, default=True, copy=False, string="Technical field, whether to create an employee")
```

Comment:
Do you really wan to create an employee for each portal user ? :upside_down_face: